benchcab/data/test/config-valid.yml

```diff
@@ -19,14 +19,13 @@ project: w97
 
 experiment: five-site-test
 
-realisations: [
-  {
-    path: "trunk",
-  },
-  {
-    path: "branches/Users/ccc561/v3.0-YP-changes",
-  }
-]
+realisations:
+  - repo:
+      svn:
+        branch_path: trunk
+  - repo:
+      svn:
+        branch_path: branches/Users/ccc561/v3.0-YP-changes
```

Comment:
@bschroeter was config-valid.yml example meant to be exhaustive in the config options or just a simple success case in the tests for schema validation? If the latter then this probably isn't necessary since the config is still valid from a validation perspective?